lib/model/folder_sendrecv.go

```diff
@@ -66,6 +66,7 @@ var (
 	errDirHasToBeScanned   = errors.New("directory contains unexpected files, scheduling scan")
 	errDirHasIgnored       = errors.New("directory contains ignored files (see ignore documentation for (?d) prefix)")
 	errDirNotEmpty         = errors.New("directory is not empty")
+	errNotAvailable        = errors.New("no peers that have this file are connected. Will be retried on next connection.")
```

Comment:
Very long, The crux here is that `no connected device has this version of the file`

Reply:
Or even "latest version of the file"